src/pages/edge/adobe-journey-optimizer/proposition-edge-events.md

```diff
@@ -0,0 +1,354 @@
+                                        "eventToken": "<BASE64_ENCODED_STRING>"
+                                    },
+                                    "activity": {
+                                        "id": "6aebeb06-faeb-4864-a27c-8be3efd6ff9b#876cb49a-7094-4f9e-9f9d-57ee26858462",
+                                        "matchedSurfaces": [
+                                            "mobileapp://com.my.app.bundleIdentifier/"
+                                        ]
+                                    },
+                                    "correlationID": "7d0408d2-e9ed-4d66-b0a5-2659fc37e2a9-0",
+                                    "decisionProvider": "AJO"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+### decisioning.propositionDismiss
+
+An event similar to the following is emitted any time a proposition is removed from the application's UI:
+
+```json
+{
+    "events": [
+        {
+            "xdm": {
+                "_id": "F8B3C334-A67D-4174-A0F2-4E00523F5D02",
+                "eventType": "decisioning.propositionDismiss",
+                "timestamp": "2024-09-25T19:48:16.092Z",
+                "_experience": {
+                    "decisioning": {
+                        "propositionEventType": {
+                            "dismiss": 1
+                        },
+                        "propositions": [
+                            {
+                                "id": "4b6fe796-3e4c-476d-be62-8610d851daef",
+                                "scope": "mobileapp://com.my.app.bundleIdentifier",
+                                "scopeDetails": {
+                                    "characteristics": {
+                                        "eventToken": "<BASE64_ENCODED_STRING>"
+                                    },
+                                    "activity": {
+                                        "id": "6aebeb06-faeb-4864-a27c-8be3efd6ff9b#876cb49a-7094-4f9e-9f9d-57ee26858462",
+                                        "matchedSurfaces": [
+                                            "mobileapp://com.my.app.bundleIdentifier/"
+                                        ]
+                                    },
+                                    "correlationID": "7d0408d2-e9ed-4d66-b0a5-2659fc37e2a9-0",
+                                    "decisionProvider": "AJO"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+### decisioning.propositionSuppressDisplay
+
+This event is emitted any time the triggering criteria are met for a message, but the SDK does not display the message.
+
+The value in the `xdm._experience.decisioning.propositionAction.reason` property is used to describe why the display was suppressed.
+
+#### Use case #1
```

Comment:
Instead of just saying "Use case 1" and "Use case 2", should we title them here to make it a bit clearer in the headers what the use cases will be?